### DISCUSSION TITLE
🚀 Pipeless 1.0 is out!

### DISCUSSION THREAD

**Discussion by miguelaeh** [miguelaeh/pipeless#73] [Announcements]
I am really happy to announce Pipeless 1.0 is out!

Find the whole article at: https://www.pipeless.ai/blog/pipeless-1-0/Pipeless%201.0:%20Serverless%20development%20experience%20for%20embedded%20computer%20vision

**Comment by shanearthur** [comment]
Congrats! Looking forward to playing around with this.

**Reply by miguelaeh** [reply]
Awesome! Please feel free to share feedback or suggestions
